daemons/fenced/fenced_commands.c

```diff
@@ -340,7 +340,8 @@ stonith_device_execute(stonith_device_t * device)
     action_limit = get_action_limit(device);
     if (action_limit > -1 && active_cmds >= action_limit) {
         crm_trace("%s is over its action limit of %d (%u active action%s)",
-                  device->id, action_limit, active_cmds, active_cmds > 1 ? "s" : "");
+                  device->id, action_limit, active_cmds,
+                  pcmk__plural_s(active_cmds));
```

Comment:
For consistency with pcmk__plural_alt, it might be nice to eventually change pcmk__plural_s to also take the word to be pluralized.  So here, it would be `pcmk__plural_s(active_cmds, "action")`.